gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -262,6 +262,42 @@ case object RandomUUID extends ElPart[UUID] {
   def apply(session: Session): Validation[UUID] = version4UUID().success
 }
 
+case object RandomInt extends ElPart[Int] {
+  def apply(session: Session): Validation[Int] = ThreadLocalRandom.current().nextInt().success
+}
+
+final case class RandomIntRange(arg: String) extends ElPart[Int] {
+
+  def randomInt: Int = {
+    val values = arg.split(",")
+    val min = values(0).toInt
+    val max = values(1).toInt
```

Comment:
See my comment below for performing this parsing with parser combinators.
Then, 2 bad things here:
* the pattern doesn't allow for white spaces, eg `randomInt(5, 10)`, which users are very likely to write
* parsing should only be done once, not on each iteration